protos/flyteidl/admin/description_entity.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package flyteidl.admin;
+option go_package = "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin";
+
+import "flyteidl/core/identifier.proto";
+import "flyteidl/admin/common.proto";
+
+// DescriptionEntity contains detailed description for the task/workflow.
+// Documentation could provide insight into the algorithms, business use case, etc.
+message DescriptionEntity {
+  // id represents the unique identifier of the description entity.
```

Comment:
sorry this is the id of the associated task/workflow right?  it's not the id of the description itself.  because the core.Identifier object does not have a resource type for a "description object".

Reply:
any reason not to add an identifier resource type for description entities?

Reply:
> sorry this is the id of the associated task/workflow right

Right. 

We store the description of the task and the workflow in the same table. `resource_type` is one of our primary key in the table, and we use it to look up the specific resource's description. So we didn't add an identifier resource type for description entities.

The table will be like
| resource_type | name | description | ... |
| ------------- | ------------- | ---------- | -------- |
task | t1 | t1 is .... | ... | ...
workflow | wf1 | wf1 is ... | ... 
... | ... | ... | ...